### PR TITLE
Cleaned UI

### PR DESCRIPTION
**Pull request by utkuozbulak**
UI Stuff

### CODE REVIEW

**Comment by utkuozbulak** [comment]
Updating

File: python-src/webapp/app.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@import url(https://fonts.googleapis.com/css?family=Abel);body {
```
Comment (andreaseliasson) [inline python-src/webapp/app.css]: Do we need to include the python-src/webapp/ dir on github?

Comment (blanche) [reply python-src/webapp/app.css]: No, we actually should add it to gitignore

Comment (blanche) [reply python-src/webapp/app.css]: No, we actually should add it to gitignore

**Review by JaneJiangHH — APPROVED** [review]
Very nice

**Pull request merged**